### ISSUE TITLE
[Suggestion] Different sound and animation for soap

### ISSUE THREAD

**Issue by Sunconure11** [squeek502/VeganOption#9]
I think its a bit weird that the player eats soap with this mod installed. Do you think it would be possible to change the animation and sounds used for it so it does not look like you are eating soap?

**Comment by squeek502** [comment]
Agreed. This is something I wanted to do but haven't looked into how to do custom item usage animations at all yet.

**Comment by Sunconure11** [comment]
Well, Thaumcraft 4 did it with their soap.

**Comment by jawsawn** [comment]
Yup, thaumcraft sanitizing soap has using animation

**Comment by elifoster** [comment]
Butchercraft soap also has its own use animation, the source for which is here: https://github.com/Lance5057/Butchercraft/blob/2cc08038017290cfdfb204bb417d21e6ce5daab7/src/main/java/com/lance5057/butchercraft/items/SoapItem.java
